agent/azure-monitor-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/SpanDataMapper.java

```diff
@@ -120,15 +120,19 @@ public TelemetryItem map(SpanData span) {
     return map(span, itemCount);
   }
 
-  public void map(SpanData span, Consumer<TelemetryItem> consumer) {
+  public void map(
+      SpanData span,
+      Consumer<TelemetryItem> consumer,
+      BiPredicate<SpanData, EventData> shouldSuppress) {
```

Comment:
can we pass this in to the constructor instead, similar to `BiPredicate<EventData, String> eventSuppressor`?

Reply:
done